feeds/src/contracts/FluxAggregatorContract.js

```diff
@@ -0,0 +1,310 @@
+import { ethers } from 'ethers'
+import {
+  getLogs,
+  formatAnswer,
+  decodeLog,
+  createContract,
+  createInfuraProvider,
+} from './utils'
+import _ from 'lodash'
+
+export default class PrepaidContract {
+  answerUpdatedEvent = {
+    filter: {},
+    listener: {},
+  }
+  submissionReceivedEvent = {
+    filter: {},
+    listener: {},
+  }
+
+  newRoundEvent = {
+    filter: {},
+    listener: {},
+  }
+
+  answerIdInterval
+  provider
+  contract
+
+  constructor(options, abi) {
+    this.provider = createInfuraProvider(options.networkId)
+    this.contract = createContract(options.contractAddress, this.provider, abi)
+    this.address = options.contractAddress
+    this.alive = true
+    this.abi = abi
+    this.options = options
+  }
+
+  kill() {
+    try {
+      if (!this.alive) return false
+      clearInterval(this.answerIdInterval)
+      this.removeListener(
+        this.answerUpdatedEvent.filter,
+        this.answerUpdatedEvent.listener,
+      )
+      this.contract = null
+      this.address = null
+      this.alive = false
+      this.options = null
+    } catch (error) {
+      //
+    }
+  }
+
+  removeListener(filter, eventListener) {
+    if (!this.alive) return
+
+    this.provider.removeListener(filter, eventListener)
+  }
+
+  async oracles() {
+    return await this.contract.getOracles()
+  }
+
+  async minimumAnswers() {
```

Comment:
Shouldn't we be using `parseInt` for `minimumAnswers`, `latestRound` and `reportingRound`? They should never return decimals

Reply:
If this is a BigNumber, there is a 'toNumber' method on it